### PR TITLE
Removed broken util get_contract_path

### PR DESCRIPTION
**Pull request by hackaugusto**
The smart contracts are not part of the raiden code anymore, the
contract manager from the raiden-contracts must be used. This removes
the util get_contract_path since it pointed to a phantom folder, and
changes the utility code to deploy from the contract manager.

### CODE REVIEW

File: raiden/utils/smart_contracts.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from eth_utils import to_canonical_address
+
+from raiden.network.rpc.client import JSONRPCClient
+from raiden.utils.typing import Address, Any, Tuple
+from raiden_contracts.contract_manager import ContractManager
+
+
+def deploy_contract_web3(
```
Comment (LefterisJP) [inline raiden/utils/smart_contracts.py]: Why was this moved inside raiden utils? It's only used in tests and not in production. The raiden client does not need to deploy any contracts.

Comment (rakanalh) [reply raiden/utils/smart_contracts.py]: @LefterisJP apparently, it's used in [here](https://github.com/raiden-network/raiden/pull/3440/files#diff-3cbaf20d13d85c8a56f8f44bfb443354R198). Which is basically for console's `ConsoleTools.create_token` method.
